elements/lisk-p2p/src/p2p.ts

```diff
@@ -647,28 +656,33 @@ export class P2P extends EventEmitter {
 	}
 
 	public async request(packet: P2PRequestPacket): Promise<P2PResponsePacket> {
-		const response = await this._peerPool.request(packet);
+		const bufferData = this._getBufferData(packet.data);
+		const response = await this._peerPool.request({ ...packet, data: bufferData });
```

Comment:
maybe its better to be specific in order to avoid any extra property, 
```suggestion
		const response = await this._peerPool.request({ procedure: packet.procedure, data: bufferData });
```